tests/TestHarness/Cluster.py

```diff
@@ -1044,6 +1045,7 @@ def activateInstantFinality(self, launcher, biosFinalizer, pnodes):
         if not self.biosNode.waitForTransFinalization(transId, timeout=21*12*3):
```

Comment:
So is the host function sent to the biosNode first even if biosFinalizer is false?

Reply:
Yes. Doesn't matter which node you look for the transaction on. Maybe we should rename this python method now. It has been there for a long time. It means is the transaction in a LIB block.